src/UI/Component/Panel/Listing/Listing.php

```diff
@@ -22,4 +22,18 @@ public function getTitle();
 	 * @return \ILIAS\UI\Component\Item\Group[]
 	 */
 	public function getItemGroups();
+
+	/**
+	 * Sets the action drop down to be displayed on the right of the title
+	 * @param \ILIAS\UI\Component\Dropdown\Standard $actions
+	 * @return Listing
+	 */
+	public function withActions(\ILIAS\UI\Component\Dropdown\Standard $actions);
```

Comment:
As @alex40724 already stated, that matches the way we do it with items we should do it the same way here. Shy Buttons are no good here since we should really leave to opportunity open here to group actions with dividers.